core/fake/mutation_storage.go

```diff
@@ -44,7 +44,12 @@ func (m *MutationStorage) ReadPage(_ context.Context, mapID, start, end int64, p
 	if end > int64(len(m.mtns[mapID])) {
 		end = int64(len(m.mtns[mapID]))
 	}
-	return end, m.mtns[mapID][start:end], nil
+	entryupdates := m.mtns[mapID][start:end]
+	entries := make([]*pb.Entry, 0, len(entryupdates))
+	for _, e := range entryupdates {
```

Comment:
Is the non-deterministic ordering from map iteration OK here?

Reply:
The top level object has a map of slices. 
This particular `range` is iterating over the slice and has deterministic ordering.

Reply:
Oops, didn't spot that.  Thanks.